config/nuxt/lib/data.js

```diff
@@ -1,6 +1,6 @@
 const appConfig = require('../../../src/client/static/data/app.json')
 
 module.exports = {
-  appConfig: { title: 'AST 2.0' },
+  appConfig: { title: appConfig.app.title },
```

Comment:
Is it guaranteed that there's a title available? If not you could provide a fallback

Reply:
Yes, this will be guaranteed.